src/HttpHelper/HttpClientHelper/HttpClientException.php

```diff
@@ -0,0 +1,5 @@
+<?php declare(strict_types=1);
+
+namespace Leadin\SurvivalKitBundle\HttpHelper\HttpClientHelper;
+
+class HttpClientException extends \RuntimeException {}
```

Comment:
Can we have this exception extends some kind of GuzzleException. This way we can stay in the same context of exception and we can always access native method from guzzle.

Reply:
no need to extend  GuzzleException, we want to create our "own" exception" in order to standardise the format of those exceptions